modules/gapi/src/compiler/gislandmodel.cpp

```diff
@@ -117,8 +117,7 @@ ade::NodeHandle GIsland::producer(const ade::Graph &g,
     }
     // Consistency: A GIsland requested for producer() of slot_nh should
     // always had the appropriate GModel node handle in its m_out_ops vector.
-    GAPI_Assert(false);
-    return ade::NodeHandle();
+    GAPI_Assert(false && "Broken GIslandModel ?.");
```

Comment:
this `return ade::NodeHandle` has been put here to avoid warnings like "non-void function does not return anything".

Can we face this issue now?

Reply:
any ideas in what environment  they were raised ?

Reply:
May be standalone debug? (BTW, "magic" standalone stuff above is broken - **please fix that**)

Reply:
it seems that standalone (both [debug](https://pullrequest.opencv.org/buildbot/builders/precommit_linux64_no_opt/builds/23414/steps/build%20gapi%20standalone) and [release](https://pullrequest.opencv.org/buildbot/builders/precommit_linux64/builds/25549/steps/build%20gapi%20standalone)) are OK